### ISSUE TITLE
Remote Web access to tuning etc?

### ISSUE THREAD

**Issue by dogastus** [tomvdb/open_tuner#52]
Would it be possible to add remote access to select a station or frequency? This is so that someone on the Internet could access a receiver remotely and use it with streaming.

**Comment by tomvdb** [comment]
Yes, it is definitely possible to add this in. What did you have in mind? A websocket to send command from a webinterface or a complete web server builtin?

**Comment by dogastus** [comment]
Just a method of setting freq and symbol rate. If there was a way of 'serving' the spectrum display over the web, a remote user could then select the desired station.

**Comment by dogastus** [comment]
You mentioned Quicktune UDP integration in the latest release but I can't make it work. Any guidlines on what settings to use in Quicktune?

**Comment by tomvdb** [comment]
> You mentioned Quicktune UDP integration in the latest release but I can't make it work. Any guidlines on what settings to use in Quicktune?

Sorry, I missed this question. Did you come right with this ? 

https://www.zr6tg.co.za/opentuner-quicktune-control/
